features/page_objects/choose_exemptions_page.rb

```diff
@@ -10,18 +10,36 @@ class ChooseExemptionsPage < SitePrism::Page
 
   element(:submit_button, "input[name='commit']")
 
-  def submit(args = {})
-    search_val = "checkbox-#{args[:exemption]}"
-
-    select_tab(args[:tab])
-
-    if args.key?(:exemption)
-      exemptions.find { |chk| chk["id"] == search_val }.click
+  # Takes each exemption number in the array
+  # Identifies what tab that exemption is on using the identify_tab method
+  # Selects that tab using the select_tab method
+  # Then clicks on the relevant checkbox for that exemption number
+  def submit(args = { exemptions: %w(S1 D2) })
```

Comment:
I'd say don't use a default value for the `args` object. I think when it comes to maintaining the project in the future it'll be clearer if you can see in the step what exemptions you're selecting, rather than just see `@app.choose_exemptions_page.submit` with no args.

Appreciate it might mean the same thing is repeated in multiple places, but I'd prefer seeing the arguments rather than wondering what has been done behind the scenes.